test/test_selector_test.rb

```diff
@@ -7,7 +7,16 @@ def setup
       target_sha = @repo.branches['change_fizz'].target.oid
       master_sha = @repo.branches['master'].target.oid
       base_sha = @repo.merge_base(target_sha, master_sha)
-      @selector = TTNT::TestSelector.new(@repo, target_sha, base_sha)
+
+      # NOTE: Use the same configuration for Rake::TestTask with fixtures/Rakefile
+      #       to create TTNT::TestTask which are consistent with actual data
+      rake_task = Rake::TestTask.new(:test) do |t|
+        t.libs << "test"
```

Comment:
I think so because `TTNT::TestSelector` expects to receive the same `TTNT::TestTask` object defined in `Rakefile`.

...and after saying that, I realized :bulb: that receiving an instance of `TTNT::TestTask` (and making such undocumented assumption) was really a bad design. So I changed `TTNT::TestSelector` to receive candidate test files rather than an instance of `TTNT::TestTask`. This way, `TTNT::TestSelector` can be instantiated with minimal data necessary for testing purpose.

Thank you for your critical comment which made me rethink about good design! :smile: 
Could you check if the approach is OK? Change is here: https://github.com/Genki-S/ttnt/commit/5b0eeb389e8d71c27818707854fac0a2ac02adab